Header_CPP.cpp

```diff
@@ -0,0 +1,96 @@
+#include "Task II - Header.h"
+int Store::getVecSize()

```

Comment:
Method names should be descriptive.  I prefer wordy, but just wordy enough.  Definitely not too wordy.  😉 

Bytes are cheap these days so we can have long identifiers.  I'd probably name `getVecSize` as `getItemCount`.